docs/source/index.rst

```diff
@@ -11,6 +11,18 @@ Features
 - Fully type annotated for great autocomplete experience
 - Supports Python 3.5 up to 3.8
 
+Release Notes
+-------------
+
+See the `Changelog`_ on GitHub.
+
+Note that this project follows `Semantic Versioning`_. Versions
+below `1.0.0` are not guaranteed to have a stable API. For this reason, please
+pay attention to the `Changelog`_.
```

Comment:
This is a good idea. I'm, however, not sure how many people might read this here. Should we also add this to the README file in the first few lines to make this clear to anyone using the SDK?

Reply:
I think I added it in the README above. Is that what you had in mind?

Reply:
Yes, I see that this is actually sourced by the README file. Thanks!